src/agent/adu_core_interface/src/adu_core_interface.c

```diff
@@ -120,7 +120,7 @@ void ADUC_WorkflowData_Uninit(ADUC_WorkflowData* workflowData)
 /**
  * @brief Reports the client json via PnP so it ends up in the reported section of the twin.
  *
- * @param messageType The message type.
+ * @param messageType The message type.`
```

Comment:
why adding back tick back? revert this file to the commit before

Reply:
I have no idea why this was changed. Checking files again to see whats going on